examples/starcoder2/qlora.yml

```
@@ -0,0 +1,70 @@
+base_model: bigcode/starcoder2-3b
+
+load_in_8bit: false
+load_in_4bit: true
+strict: false
+
+datasets:
+  - path: mhenrichsen/alpaca_2k_test
+    type: alpaca
+
+chat_template: chatml
+
+dataset_prepared_path: dataset_prepared
```

```suggestion
dataset_prepared_path:
```

Should this be empty?